src/main/java/core/basesyntax/FiguresName.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum FiguresName {
+    CIRCLE,
+    SQUARE,
+    RIGHT_TRIANGLE,
+    RECTANGLE,
+    ISOSCELES_TRAPEZOID
+}
```

Comment:
What if we will create another `Figure` subclass? Will we always update this enum accordingly?

Let's try not to use it. See comments tips in `FigureSupplier`